keps/sig-storage/625-csi-migration/README.md

```diff
@@ -194,12 +194,31 @@ you need any help or guidance.
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [x] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: CSIMigration, CSIMigration{cloud-provider}
+    - Feature gate name: CSIMigration, CSIMigration{cloud-provider}, InTreePlugin{cloud-provider}Unregister
     - Components depending on the feature gate: kubelet, kube-controller-manager, kube-scheduler
     - Please refer to this design doc on the [Step to enable the feature](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md#upgradedowngrade-migrateunmigrate-scenarios)
 
 * **Does enabling the feature change any default behavior?**
-  Yes and No. If only `CSIMigration` feature flag is enabled, nothing will change on the cluster behavior. However, if `CSIMigration` && `CSIMigration{cloud-provider}` are both enabled, the behavior will change. The in-tree volume plugin that the cloud-provider use will be redirect to use the corresponding CSI driver. But from a user perspective, nothing will be noticed.
+
+  Yes and No.
+  - If only `CSIMigration` feature flag is enabled, nothing will change on the cluster behavior. `CSIMigration`
+  is a big umbrella feature gate. `CSIMigration{cloud-provider}` will not take any affect without it.
+  - If only `CSIMigration{cloud-provider}` feature flag is enabled, nothing will change on the cluster behavior.
+  - `InTreePlugin{cloud-provider}Unregister` is a standalone feature gate that can be enabled and disabled
+  even out of CSI Migration scope. The name speaks for itself, when enabled, the controller will not 
+  register the specific in-tree storage plugin to the supported list. If User only enables this flag, they
+  will get an error from PVC saying it cannot find the plugin.
+  - If `CSIMigration` && `CSIMigration{cloud-provider}` are both enabled on Kubelet and KCM, the behavior will 
+  change. The in-tree volume plugin that the cloud-provider use will be redirect to use the corresponding CSI driver
+  depending on the operation types and if the flag is enabled on both Kubelet and KCM. But from a user perspective, 
+  nothing will be noticed.
+  - If `CSIMigration` && `CSIMigration{cloud-provider}` are enabled only on KCM, operations like Provision/Deletion 
+  will be redirect to CSI driver. Other operations will still go with the in-tree plugin.
+  - If `CSIMigration` && `CSIMigration{cloud-provider}` are enabled only on Kubelet, this will end up with a broken
+  state. Operations like volume provision will still work. But operations like volume Attach/Mount will be broken.
+  - If all `CSIMigration`, `CSIMigration{cloud-provider}` and `InTreePlugin{cloud-provider}Unregister` are
+  enabled at the same time, the in-tree volume plugin that the cloud-provider use will be redirect to use 
+  the corresponding CSI driver regardless whether `CSIMigration{cloud-provider}` is enabled for Kubelet or not.
```

Comment:
This bulleted list is hard to follow, can you turn it into a table? e.g.

| - | CSIMigration=enabled | CSIMigration=disabled |
| -- | -- | -- |
| **CSIMigration{cloud-provider}=enabled** | <ul><li><b>InTreePlugin{cloud-provider}Unregister=enabled</b> ABC</li><li><b>InTreePlugin{cloud-provider}Unregister=disabled</b> XYZ</li></ul> | etc. |

Reply:
I updated it with a table. Please take a look. Thanks!